### PR TITLE
Burn otoken functionality

### PR DESCRIPTION
**Pull request by haythem96**
# Task: Feature Name:

Controller: burn otoken 

## High Level Description

This PR add the functionality of burning un-expired short otoken.

### Code

- [x] Unit test 100% coverage
- [x] Does your code follow the naming and code documentation guidelines?

### Documentation

- [x] Is your code up to date with the spec? 
- [x] Have you added your tests to the testing doc?

### CODE REVIEW

**Review by aparnakr — APPROVED** [review]
Looks good to merge! 😄

File: contracts/Controller.sol [hunk 1]
```diff
@@ -93,6 +93,14 @@ contract Controller is ReentrancyGuard, Ownable {
         uint256 vaultId,
         uint256 amount
     );
+    /// @notice emits an event when a short otoken get burned from a vaukt
+    event ShortOtokenBurned(
```
Comment (antoncoding) [inline contracts/Controller.sol:97]: Can we just call it OtokenBurned? (not important)

File: test/controller.test.ts [hunk 2]
```diff
@@ -1981,6 +1981,323 @@ contract('Controller', ([owner, accountOwner1, accountOperator1, random]) => {
...
+        await shortOtoken.transfer(accountOwner1, operatorShortBalance, {from: accountOperator1})
+
+        const vaultCounter = new BigNumber(await controller.getAccountVaultCounter(accountOwner1))
+        assert.isAbove(vaultCounter.toNumber(), 0, 'Account owner have no vault')
+
+        const vaultBefore = await controller.getVault(accountOwner1, vaultCounter)
+
+        const shortOtokenToBurn = new BigNumber(vaultBefore.shortAmounts[0])
+        console.log(shortOtokenToBurn.toString())
+        console.log(new BigNumber(await shortOtoken.balanceOf(accountOwner1)).toString())
```
Comment (antoncoding) [inline test/controller.test.ts:2162]: Remove this

File: test/controller.test.ts [hunk 3]
```diff
@@ -1981,6 +1981,323 @@ contract('Controller', ([owner, accountOwner1, accountOperator1, random]) => {
...
+          new BigNumber(vaultBefore.shortAmounts[0]).minus(new BigNumber(vaultAfter.shortAmounts[0])).toString(),
+          shortOtokenToBurn.toString(),
+          'Short otoken amount in vault after burn mismatch',
+        )
+      })
+
+      describe('Burn expired otoken', () => {
+        let expiredShortOtoken: MockOtokenInstance
+
+        before(async () => {
```
Comment (antoncoding) [inline test/controller.test.ts:2199]:
```suggestion
        before('mint otoken before expiry',async () => {
```

**Review by antoncoding — APPROVED** [review]
Delete the console.log and we're good

**Pull request merged**